examples/rest-crud/README.md

```diff
@@ -0,0 +1,74 @@
+# @loopback/example-rest-crud
+
+This is a basic example demonstrating how to use default CRUD REST repository
```

Comment:
> This is a basic example demonstrating how to use default CRUD REST repository
and controller with a single model class.

This is a basic example demonstrating how to use the default CRUD REST repository
and controller with a single model class.